src/routes/chat/Chat.jsx

```diff
@@ -34,10 +35,22 @@ const ChatContent = () => {
 };
 
 const Chat = () => {
+  const dispatch = useDispatch();
   const { chatRoomId } = useParams();
   const correspondentName = useSelector(
```

Comment:
Repeating myself from the other review: we don't need this variable now since we don't have the dealerName anymore.

Reply:
Wanted to fix this in next PR
But ok

Reply:
I wanted to resolve this in the next PR
But ok